src/pages/sighting/encounters/AnnotationsCard.jsx

```diff
@@ -61,54 +61,56 @@ export default function AnnotationsCard({
         sightingData={sightingData}
       />
       {annotationReferences.length > 0 ? (
-        <Grid container spacing={2}>
-          {annotationReferences.map(annotationReference => {
-            const matchingAsset = assets.find(
-              asset =>
-                asset?.guid === annotationReference?.asset_guid,
-            );
-            const matchingAssetAnnotations = get(
-              matchingAsset,
-              'annotations',
-              [],
-            );
-            const matchingAnnotation = matchingAssetAnnotations.find(
-              annotation =>
-                annotation?.guid === annotationReference?.guid,
-            );
-            return (
-              <Grid
-                key={matchingAnnotation?.guid}
-                item
-                style={{ position: 'relative' }}
-              >
-                <AnnotatedPhotograph
-                  assetMetadata={matchingAsset}
-                  annotations={[matchingAnnotation]}
-                  onClick={Function.prototype}
-                  width={248}
-                />
-                <IconButton
-                  onClick={e =>
-                    setAnchorInfo({
-                      element: e.currentTarget,
-                      annotation: matchingAnnotation,
-                    })
-                  }
-                  style={{
-                    position: 'absolute',
-                    top: 12,
-                    right: 12,
-                    color: theme.palette.common.white,
-                  }}
-                  className={classes.photoIcon}
+        <div style={{ margin: '8px 0' }}>
```

Comment:
surrounding the Grid in a div with 8px margin on top and bottom cancels out the -8px from material-ui

Reply:
I think the style could have been added directly to the Grid and saved you one div.

Reply:
that works! changed